presto-product-tests/README.md

```diff
@@ -143,6 +143,12 @@ presto-product-tests/bin/run_on_docker.sh <profile> -t com.facebook.presto.tests
 presto-product-tests/bin/run_on_docker.sh <profile> -t sql_tests.testcases.system.selectInformationSchemaTables
 ```
 
+Profile specific tests are tests which runs only for that particular profile. It confirms if the profile is same as what is claims.
+For e.g. singlenode-kerberos-hdfs-impersonation - Test to check if kerberos is working with impersonation
```

Comment:
Do not use "For e.g." as a shortcut for "For example", it is too colloquial and grammatically incorrect. Just say "For example".